src/utils/transformers/index.ts

```diff
@@ -178,21 +192,21 @@ export const elementsMap = {
   },
   svg: {
     allowedAttributes: ALLOWED_ATTRIBUTES.svg,
-    postProcessAttributes: (attributes, props, node) => {
+    postProcessAttributes: (attributes?: Attributes, props?: Props, node?: XmlNode) => {
       const { height, width } = props || {};
 
       if (!height && !width) {
         return attributes;
       }
 
-      const attrs = Object.assign({}, attributes);
+      const attrs = Object.assign({} as Partial<Props>, attributes);
 
-      if (height) {
-        attrs.height = height;
+      if (height !== undefined) {
+        attrs.height = height?.toString();
       }
 
-      if (width) {
-        attrs.width = width;
+      if (width !== undefined) {
+        attrs.width = width?.toString();
       }
```

Comment:
This is a bit redundant. You don't need the `ifs` AND the optional chaining:
```ts
attrs.height = height?.toString() ?? attrs.height;
attrs.width = width?.toString() ?? attrs.width;
```

Reply:
Good catch!